spanner/ddl/ast.go

```diff
@@ -494,6 +508,23 @@ func (k Foreignkey) PrintForeignKeyAlterTable(spannerSchema Schema, c Config, ta
 	return s
 }
 
+// PrintCheckConstraintTable formats the check constraints in SQL syntax.
+func PrintCheckConstraintTable(cks []CheckConstraint) string {
```

Comment:
nit: this is not really printing the check constraint.

Suggest this be renamed to formatCheckConstraints or something similar

Reply:
function name renamed to "FormatCheckConstraints"